release_notes.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+pip install git+https://github.com/uc-cdis/release-helper.git@master#egg=gen3git
```

Comment:
perhaps versioneer? Looks like this would be more multi-language though.

Reply:
Yeah, I've looked into versioneer, it seems out of maintenance for over a year, and it's adding quite some more extra stuff (setup.cfg and _version.py), yet I don't know how to fit into [the requirement](https://github.com/uc-cdis/dictionaryutils/pull/37#discussion_r257898408).

Oh, but did you mean that versioneer may generate release notes too? I guess people might already get used to the PR format that our custom release helper supports, so it affects more than a different implementation unless it has a greater advantage.